packages/react-server/core/ReactServerAgent.js

```diff
@@ -10,8 +11,14 @@ function makeRequest (method, url) {
 	return new Request(method, url, API.cache());
 }
 
+const DATA_BUNDLE_CACHE     = {};
```

Comment:
Heh.  Good call.  Pushed a patch that both adds a comment and also just makes `DATA_BUNDLE_CACHE` undefined on the server.